lib/logger.js

```diff
@@ -11,8 +11,44 @@ Hoek.assert(process.env.LOG_LEVEL, 'Must define LOG_LEVEL');
 var stream = new PrettyStream();
 stream.pipe(process.stdout);
 
+function requestSerializer(req) {
+  return {
+    path: req.path,
+    method: req.method,
+    headers: req.headers
+  };
+}
+
+function responseSerializer(res) {
+  return {
+    payload: res.output.payload
+  };
+}
+
+function errorSerializer(res) {
+  if (!res.data) {
+    return;
+  }
+
+  if (typeof res.data.error === 'string') {
+    return {
+      message: res.data.error
+    };
+  }
+
+  var serializedData = JSON.parse(JSON.stringify(res.data.error));
+  serializedData.message = res.data.error.message;
```

Comment:
Remember how some attributes weren't being logged because they weren't enumerable? `res.data.error.message` is one of them, so I explicitly call it out here to make sure we see it.